packages/types/src/interfaces/augment-api-query.ts

```diff
@@ -100,13 +60,13 @@ declare module '@polkadot/api/types/storage' {
        **/
       uncles: AugmentedQuery<ApiType, () => Observable<Vec<UncleEntryItem>>> & QueryableStorageEntry<ApiType>;
     };
+    // @ts-ignore
     babe: {
```

Comment:
I don't know why it add babe here, when babe already exist.. investigating this.. add @ts-ignore for now

Reply:
I think the reason of creation of this field is -
a. In node (CENNZnet) the state of Babe is little behind to what is available with substrate.. i.e authorVrfRandomness does not exist in ours..
b. Polkadot's latest version takes into account that..
For more information check attached file..
<img width="1791" alt="Screen Shot 2020-11-30 at 2 09 13 PM" src="https://user-images.githubusercontent.com/29415595/100559062-f0aafe80-3315-11eb-9c82-482aa5d451eb.png">

Reply:
So what does ts-ignore doing here? what is ignored?

Reply:
We are ignore typescript error.. 
TS2717: Subsequent property declarations must have the same type. Property 'babe' must be of type '{ [key: string]: QueryableStorageEntry<ApiType>; authorities: MethodResult<ApiType, () => Observable<Vec<ITuple<[AuthorityId, BabeAuthorityWeight]>>>> & StorageEntryBase<...> & QueryableStorageEntry<...>; ... 10 more ...; underConstruction: MethodResult<...> & ... 1 more ... & QueryableStorageEntry<...>; }', but here has type '{ [key: string]: QueryableStorageEntry<ApiType>; authorities: MethodResult<ApiType, () => Observable<Vec<ITuple<[AuthorityId, BabeAuthorityWeight]>>>> & StorageEntryBase<...> & QueryableStorageEntry<...>; ... 9 more ...; underConstruction: MethodResult<...> & ... 1 more ... & QueryableStorageEntry<...>

Polkadot's type generation script uses Module Augmentation so when it has its own definition of Babe, it shouts with ts error